doc/Alerting/Transports.md

```diff
@@ -46,6 +46,7 @@ The one and only possible parameter to be passed is `source` - this is required
 | ------ | ------- |
 | Alertmanager URL      | http://alertmanager.example.com |
 | Alertmanager Options: | source=librenms |
```

Comment:
As you removed `source` in Alertmanager.php, do you still need to set it up?

Reply:
Depends on the setup a user has, really. Initially it appeared logical to me (!) that one would like to specify source as one of the labels. Then I stumbled upon a requirement to add two labels which would not be usable for majority of users (mine will be source and escalation number), so ended up offering user to specify whatever he wants. Code block at line 68 does it. 
So in end, if it is "source" user wants - he can use "source". If it is something else - it can be something else.